custom_event_checks/opslevel/service_infrastructure_check/aws-rds-certificate-expiration-check.yml

```
@@ -0,0 +1,73 @@
+---
+version: 1
+kind: generic
+spec:
+  name: AWS RDS Certificate Expiration Check
+  serviceSelector: ".[] | .aliases[0]"
+  successCondition: '[.[] | select(.aliases[0] == $ctx.alias) | .relatedResources.nodes[]
+    | select(.type == "Database") | .rawData | (.ca_certificate_identifier == "rds-ca-rsa2048-g1"
+    or .ca_certificate_identifier == "rds-ca-rsa4096-g1" or .ca_certificate_identifier
+    == "rds-ca-ecc384-g1")]| all'
+  message: |-
+    {% if check.passed %}
+      ### Check passed
+
+    Service **{{ctx.alias}}** has all databases up to date.
+
+    {% for service in data %}
+    {% if service.aliases[0] == ctx.alias %}
+    {% for infra in service.relatedResources.nodes %}
+    {% if infra != empty %}
+    {% if infra.type == "Database" %}
+    Infrastructure Resource: **{{ infra.name }}**
+    * ca_certifate_identifier: {{ infra.rawData["ca_certificate_identifier"] }}
+    * http://app.opslevel.com/{{ infra.href }}
+    {% endif %}
+    {% endif %}
+    {% endfor %}
+    {% endif %}
+    {% endfor %}
+
+    {% else %}
+      ### Check failed
+      Service **{{ ctx.alias }}** has failed. Please see the list of infrastructure resources below and update databases that still have `rds-ca-2019`. The correct certificate is `rds-ca-rsa2048-g1`
+
+    {% for service in data %}
+    {% if service.aliases[0] == ctx.alias %}
+    {% for infra in service.relatedResources.nodes %}
+    {% if infra != empty %}
+    {% if infra.type == "Database" and infra.rawData["ca_certificate_identifier"] == "rds-ca-2019" %}
+    Infrastructure Resource: **{{ infra.name }}**
+    * ca_certifate_identifier: {{ infra.rawData["ca_certificate_identifier"] }}
+    * http://app.opslevel.com/{{ infra.href }}
```

```suggestion
    * http://app.opslevel.com{{ infra.href }}
```

I know it works but i think the extra slash here is unnecessary - you can even see how it generates in the image you linked.

---

Great catch, made the change!